### ISSUE TITLE
Make Waffle addressable as if it were a 2D list

### ISSUE THREAD

**Issue by lawsie** [lawsie/guizero#126]
I forgot all about this! I was going to make the waffle pixels addressable as if they were a 2D list.

e.g. `waffle[0][1] = "red"`

I think you do this by overriding some magic methods somehow, but it was a long time ago now!

**Comment by Coal0** [comment]
Did this in [PR 128](https://github.com/lawsie/guizero/pull/128).

**Comment by martinohanlon** [comment]
Included in 0.4.5.

You get a `WafflePixel` object back from:

```
waffle[0,0]
```

docs need updating

**Comment by martinohanlon** [comment]
Docs updated.

All pixels in the waffle are now individually "update-able" so you can do this:

```
waffle[0,0].dotty = True
```

and just change 1 pixel to be a circle.

or change its color

```
waffle[0,0].color = "red"
waffle[0,0].color = "#ff0000"
waffle[0,0].color = (255,0,0)
```

You can also gain a reference to a pixel now as well.

```
my_pixel = waffle[0,0]
```

Its in dev and will be released with 0.4.5.
